sam-styles/packages/components/button/styles/button.scss

```diff
@@ -31,6 +31,7 @@ $button-shadow: 0 0.25rem 0.5rem 0 rgba(0, 0, 0, 0.2);
     @include u-text($theme-link-color);
     border: none;
     text-decoration: none;
+    min-width: 0px !important;
```

Comment:
Just checking, can this style be applied without !important?

Reply:
no, because usa button enforces min-width on all buttons.